models/order.js

```diff
@@ -68,6 +70,18 @@ orderSchema.statics.updateStatus = function(orderId, status) {
   });
 }
 
+orderSchema.statics.setClosed = function(orderId, transactionId) {
+  const status = constants.CLOSED;
+  const transaction = transactionId;
+  return this.findOneAndUpdate(
+    { id: orderId },
+    { status, transaction },
+    { new: true }
+  )
+    .then((order) => populateEntries(toOrder(order)))
+    .then(populateTransaction);
```

Comment:
so right now the flow to have an order with transaction data is to go through the closing flow, right? would we need to do populate entries at other places to get the same info, say when someone goes from the orders page to a closed order?